examples_tests/61.BlitFilterTest/default_compute_normalization.comp

```diff
@@ -0,0 +1,192 @@
+#version 460 core
+
+#include <nbl/builtin/glsl/macros.glsl>
+
+#ifndef _NBL_GLSL_WORKGROUP_SIZE_X_
+#define _NBL_GLSL_WORKGROUP_SIZE_X_ 8
+#endif
+
+#ifndef _NBL_GLSL_WORKGROUP_SIZE_Y_
+#define _NBL_GLSL_WORKGROUP_SIZE_Y_ 8
+#endif
+
+#ifndef _NBL_GLSL_WORKGROUP_SIZE_Z_
+#define _NBL_GLSL_WORKGROUP_SIZE_Z_ 8
+#endif
```

Comment:
i'd remove these and declare them inside your `_NBL_GLSL_BLIT_NORMALIZATION_DIM_COUNT_` switch

for 1D you want 256x1x1
2D 16x16x1
3D 8x8x4

Reply:
Resolved.